operator/pkg/controllers/master/auditlog.go

```diff
@@ -0,0 +1,56 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package master
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/awslabs/kubernetes-iteration-toolkit/operator/pkg/apis/controlplane/v1alpha1"
+	"github.com/awslabs/kubernetes-iteration-toolkit/operator/pkg/utils/object"
+	v1 "k8s.io/api/core/v1"
+)
+
+func (c *Controller) reconcileAuditLogConfig(ctx context.Context, controlPlane *v1alpha1.ControlPlane) error {
+	providerConfig := auditLogConfig
```

Comment:
Does `GenerateConfigMap` mutate the config arg passed? If not, can we directly use auditLogConfig?

Reply:
Looks like it returns a ConfigMap and does not mutate the passed string. Updated to use `auditLogConfig` directly.